src/controllers/bulk/bulkcontroller.h

```diff
@@ -56,6 +56,7 @@ class BulkController : public Controller {
 
     virtual bool savePreset(const QString fileName) const;
 
+    virtual void visit(const KeyboardControllerPreset* preset);
```

Comment:
the code around the visit functions is hard to track because of the overload versions. 
My idea was to rename them to (or better names):

visitKeyboard()
visitMidi()
visitHid()

Maybe we can see clear later how to avoid the empty dummies  with the warning message